ai-ml/jupyterhub/main.tf

```diff
@@ -0,0 +1,34 @@
+#---------------------------------------------------------------
+# EKS Cluster
+#---------------------------------------------------------------
+
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "~> 19.9"
+
+  cluster_name    = local.name
+  cluster_version = var.eks_cluster_version
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = module.vpc.private_subnets
```

Comment:
Refer to spark operator blueprint for EKS Cluster and node group creation with Secondary CIDR range https://github.com/awslabs/data-on-eks/blob/47076477f1d691c0aaa7ca4a04ac215bd7e91837/analytics/terraform/spark-k8s-operator/main.tf#L17